<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="YY226163" name="www.testng.com.Testng_expectedexception" tests="3" failures="0" timestamp="2022-05-27T15:18:05 IST" time="0.032" errors="0">
  <testcase name="aMethod" time="0.024" classname="www.testng.com.Testng_expectedexception"/>
  <system-out/>
  <testcase name="cMethod" time="0.006" classname="www.testng.com.Testng_expectedexception"/>
  <system-out/>
  <testcase name="bMethod" time="0.002" classname="www.testng.com.Testng_expectedexception"/>
  <system-out/>
</testsuite> <!-- www.testng.com.Testng_expectedexception -->
